diff --git a/node_modules/expo-system-ui/plugin/src/withAndroidRootViewBackgroundColor.ts b/node_modules/expo-system-ui/plugin/src/withAndroidRootViewBackgroundColor.ts
index 69c8eac..c3c62ce 100644
--- a/node_modules/expo-system-ui/plugin/src/withAndroidRootViewBackgroundColor.ts
+++ b/node_modules/expo-system-ui/plugin/src/withAndroidRootViewBackgroundColor.ts
@@ -7,7 +7,7 @@ import {
 } from 'expo/config-plugins';
 
 const { assignColorValue } = AndroidConfig.Colors;
-const { assignStylesValue, getAppThemeGroup } = AndroidConfig.Styles;
+const { assignStylesValue } = AndroidConfig.Styles;
 
 const ANDROID_WINDOW_BACKGROUND = 'android:windowBackground';
 const WINDOW_BACKGROUND_COLOR = 'activityBackground';
@@ -32,7 +32,10 @@ export const withRootViewBackgroundColorStyles: ConfigPlugin = (config) => {
   return withAndroidStyles(config, async (config) => {
     config.modResults = assignStylesValue(config.modResults, {
       add: !!getRootViewBackgroundColor(config),
-      parent: getAppThemeGroup(),
+      parent: {
+        name: 'Theme.AppCompat.Light.NoActionBar',
+        parent: 'Theme.AppCompat.Light',
+      },
       name: ANDROID_WINDOW_BACKGROUND,
       value: `@color/${WINDOW_BACKGROUND_COLOR}`,
     });
